src/app/content/hooks/locationChange/index.ts

```diff
@@ -24,6 +25,7 @@ const hookBody: RouteHookBody<typeof content> = (services) => {
       loadStudyGuides(services)(),
       loadPracticeQuestions(services)(),
       initializeIntl(services)(),
+      loadStoredTextSize(services)(),
```

Comment:
you only have to do this here if your hook relies on the book data being loaded, i think in your case you could compose the hook directly in `src/app/content/hooks/index.ts`